.github/taskIdbyowner.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+
+codeowners_file_path = '/workspaces/azure-pipelines-tasks/.github/CODEOWNERS'
+team_name = '@microsoft/azure-artifacts-packages'
+
+def extract_tasks_owned_by_team(file_path, team):
+    tasks = []
+    with open(file_path, 'r') as file:
+        for line in file:
+            if team in line:
+                task_path = line.split()[0]
+                tasks.append(task_path)
+    return tasks
+
+def extract_task_id(task_path):
+    task_json_path = os.path.join(task_path, 'task.json')
```

Comment:
Because these paths are relative, you're making an assumption here that the script is being run from the root of the repository. It would be better to join the root of the repository, then the task_path, then task.json